bundles/org.eclipse.swt/Eclipse SWT Drag and Drop/gtk/org/eclipse/swt/dnd/TextTransfer.java

```diff
@@ -42,9 +42,11 @@ public class TextTransfer extends ByteArrayTransfer {
 	private static final String COMPOUND_TEXT = "COMPOUND_TEXT"; //$NON-NLS-1$
 	private static final String UTF8_STRING = "UTF8_STRING"; //$NON-NLS-1$
 	private static final String STRING = "STRING"; //$NON-NLS-1$
+	private static final String TEXTPLAINUTF8 = "text/plain;charset=utf-8"; //RFC-1341
```

Comment:
A nitpick but would you please follow the code style in the file  aka TEXT_PLAIN_UTF8

Reply:
Certanly I assumed that also meant TEXTPLAINUTF8_ID --> TEXT_PLAIN_UTF8_ID
Force pushed